code/world/initialization/2_init.dm

```diff
@@ -155,6 +154,7 @@
 	Z_LOG_DEBUG("World/Init", "Setting up random rooms...")
 	buildRandomRooms()
 	makepowernets()
+	build_camera_network()
```

Comment:
I believe, the build_camera_network() being inside the NO_RANDOM_ROOMS if, means that if the random rooms are disabled, no camera network will be built at all, which is not ideal if you are running quick tests.

Reply:
thank you for catching this